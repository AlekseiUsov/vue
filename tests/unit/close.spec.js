import { mount } from "@vue/test-utils";
import close from "../../src/components/atoms/ToDoCloseBth.vue";

describe("ToDoCloseBth.vue", () => {
  it("тестируем отрисовку компанента", async () => {
    const wrapper = mount(close);

    expect(wrapper).toMatchSnapshot();
  });

  it("тестируем кнопку закрывания", async () => {
    const wrapper = mount(close);

    await wrapper.find("span").trigger("click");
    expect(wrapper.emitted("close")).toBeTruthy();
  });
});
