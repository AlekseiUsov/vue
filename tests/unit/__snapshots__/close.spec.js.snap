// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDoCloseBth.vue тестируем отрисовку компанента 1`] = `<div><span></span></div>`;
